src/B2_build_movie_db.sql

```diff
@@ -0,0 +1,164 @@
+    website     VARCHAR(255)    NULL,
+    awards      VARCHAR(255)    NULL
+);
+
+CREATE TABLE season (
+    media_id        INTEGER     PRIMARY KEY,
+    "status"        VARCHAR(50) NOT NULL,
+    season_number   INTEGER     NOT NULL,
+    end_date        DATE        NULL,
+    series_id       INTEGER     NOT NULL REFERENCES media(media_id),
+    FOREIGN KEY (media_id) REFERENCES media(media_id)
+);
+
+CREATE TABLE episode (
+    media_id        INTEGER PRIMARY KEY,
+    episode_number  INTEGER NOT NULL,
+    season_id       INTEGER NOT NULL REFERENCES media(media_id),
+    FOREIGN KEY (media_id) REFERENCES media(media_id)
+);
+
+CREATE TABLE media_genre (
+    "name"      VARCHAR(50) NOT NULL,
+    media_id    INTEGER     NOT NULL REFERENCES media(media_id),
+    PRIMARY KEY (media_id, "name")
+);
+
+CREATE TABLE media_production_country (
+    media_production_country_id INTEGER     PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    media_id                    INTEGER     NOT NULL REFERENCES media(media_id),
+    country_code                VARCHAR(2)  NOT NULL REFERENCES country(code)
+);
+
+CREATE TABLE score (
+    score_id        INTEGER     PRIMARY KEY,
+    source          INTEGER     NOT NULL,
+    "value"         VARCHAR(20) NOT NULL,
+    "at"            TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (score_id) REFERENCES media(media_id)
+);
+
+CREATE TABLE "collection" (
+    collection_id   INTEGER     PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    "name"          VARCHAR(50) NOT NULL,
+    "description"   TEXT        NULL
+);
+
+CREATE TABLE media_in_collection (
+    collection_id   INTEGER REFERENCES "collection"(collection_id),
+    media_id        INTEGER REFERENCES media(media_id),
+    PRIMARY KEY (collection_id, media_id)
+);
+
+CREATE TABLE related_media (
+    primary_id    INTEGER     NOT NULL REFERENCES media(media_id),
+    related_id    INTEGER     NOT NULL REFERENCES media(media_id),
+    "type"        VARCHAR(50) NOT NULL,
+    PRIMARY KEY (primary_id, related_id)
+);
+
+CREATE TABLE release (
+    release_id      INTEGER     PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    title           VARCHAR(50) NOT NULL,
+    release_date    DATE        NULL,
+    country_code    VARCHAR(2)  NOT NULL REFERENCES country(code), -- Region
+    media_id        INTEGER     NOT NULL REFERENCES media(media_id) 
+);
+
+CREATE TABLE spoken_language (
+    spoken_language_id  INTEGER     PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    release_id          INTEGER     NOT NULL REFERENCES release(release_id),
+    language_code       VARCHAR(2)  NOT NULL REFERENCES "language"(code)
+);
+
+CREATE TABLE promotional_media (
+    promotional_media_id    INTEGER         PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    release_id              INTEGER         NOT NULL REFERENCES release(release_id),
+    "type"                  VARCHAR(50)     NOT NULL,
+    "uri"                   VARCHAR(255)    NOT NULL
```

Comment:
Couldn't we have a scenario where a promotional media, do not have an uri? Or do we want to force that a promotional media should always have an uri?

Reply:
Isn't it kinda the point of promotional media? If it has no `uri` why would it even exist?

Reply:
Well I suppose it is going to depend on the `type` of promotional media that we decide to support. I am okay with us deciding that promotional media, must always have an website in order to be considered promotional media in our system. Just wanted to make sure it was not an oversight. :)